src/components/BackButton.tsx

```diff
@@ -24,9 +24,17 @@ const BackButton = ( { label = 'Cancel', ...props }: BackButtonProps ) => {
 
   const goBack = () => navigation.goBack()
 
+  const render = () => {
```

Comment:
Do not have a render function, and instead below, use 
```
{label instanceof String 
  ? label
  : <text blah >}
```

That way you can avoid an unnecessary function, and avoid the necessity of an if block